weather-app/index.js

```diff
@@ -0,0 +1,20 @@
+const {getGeoData} = require('./helpers/geoData.js')
+const {getGeoWeather} = require('./helpers/geoWeather.js')
+
+const location = 'Grodno'
+
+getGeoData(location, (error, {longiTude, latiTude, placeName}) => {
+    if (error) {
+        console.log('error')
```

Comment:
won't work 
 - first of all you should console.log `error` variable, not a string
 - the second one it that code will move forward after getting error and won't exit from function